README.MD

```diff
@@ -1,166 +1,196 @@
+    echo -n '{"text":"my phone number is 057-555-2323 and my credit card is 4961-2765-5327-5913", "AnalyzeTemplateId":"<my-analyze-template-name>", "AnonymizeTemplateId":"<my-anonymize-template-name>"  }' | http <api-service-address>/api/v1/projects/<my-project>/anonymize
     ```
 
 ***Sample 5 (Image anonymization)***
 
-1. Create an anonymizer image template (This template redact values with black color)
+1. Create an anonymizer image template (This template redacts values with black color):
+
     ```sh
-    $ echo -n '{"fieldTypeGraphics":[{"graphic":{"fillColorValue":{"blue":0,"red":0,"green":0}}}]}' | http <api-service-address>/api/v1/templates/<my-project>/anonymize-image/<my-anonymize-image-template-name>
+    echo -n '{"fieldTypeGraphics":[{"graphic":{"fillColorValue":{"blue":0,"red":0,"green":0}}}]}' | http <api-service-address>/api/v1/templates/<my-project>/anonymize-image/<my-anonymize-image-template-name>
     ```
 
-2. Anonymize image
+2. Anonymize image:
+
     ```sh
-    $ http -f POST <api-service-address>/api/v1/projects/<my-project>/anonymize-image detectionType='OCR' analyzeTemplateId='<my-analyze-template-name>' anonymizeImageTemplateId='<my-anonymize-image-template-name>' imageType='image/png' file@~/test-ocr.png > test-output.png
+    http -f POST <api-service-address>/api/v1/projects/<my-project>/anonymize-image detectionType='OCR' analyzeTemplateId='<my-analyze-template-name>' anonymizeImageTemplateId='<my-anonymize-image-template-name>' imageType='image/png' file@~/test-ocr.png > test-output.png
     ```
 
-### Current Features Status
+## Single click deployment using the default values
+
+The script will install Presidio on your Kubenetes cluster. Prerequesites:
+
+1. A Kubernetes cluster.
+2. `kubectl` installed
+    * verify you can communicate with the cluster by running:
+
+        ``` sh
+        kubectl version
+        ```
+
+3. Local `helm` client.
+4. Recent presidio repo is cloned on your local machine.
+
+### Installation Steps
+
+1. Navigate into `<root>\deployment` from command line.
+
+2. If You have helm installed, but havn't run `helm init`, execute [deploy-helm.sh](deploy-helm.sh) in the command line. It will install `tiller` (helm server side) on your cluster.
+
+3. Grant the Kubernetes cluster access to the container registry
+    * If using Azure Kubernetes Service, follow these instructions to [grant the AKS cluster access to the ACR.](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-auth-aks)
+
+4. If you already have `helm` and `tiller` configured, or if you installed it in the previous step, execute [deploy-presidio.sh](deploy-presidio.sh) in the command line as follows:
+
+```sh
+deploy-presidio.sh <registry>
```

Comment:
```suggestion
deploy-presidio.sh
```

Reply:
`deploy-presidio.sh` needs to be updated as well to use MCR. It looks like it should have been changed by a different task